### PR TITLE
ST-1400: Change textarea counter

### PR DESCRIPTION
**Pull request by ilyawzrd**
Description:

--

QA-test: ST-1402

Demo-test:

Download artifact URL: https://vc3prerelease.blob.core.windows.net/packages/vc-theme-b2b-vue-1.4.0-pr-212-cfb7712a.zip

### CODE REVIEW

**Review by Andrew-Orlov — APPROVED** [review]

File: client-app/ui-kit/components/atoms/text-area/vc-text-area.vue [hunk 1]
```diff
@@ -9,7 +9,7 @@
       class="rounded w-full leading-tight p-3 border focus:border-gray-400 outline-none"
       :class="$attrs.class"
     />
-    <p v-if="counter" class="absolute text-xs text-gray-300 bottom-3.5 right-3">{{ symbolsCount }}</p>
+    <p v-if="counter" class="text-xs text-[color:var(--color-danger)] self-end">{{ symbolsCount }}</p>
```
Comment (Flanker72) [inline client-app/ui-kit/components/atoms/text-area/vc-text-area.vue:12]: Maybe it's not good to always show counter with danger color.
Please, make this logic instead:
- show counter in gray by default
- show counter in danger color if there is less then 10 symbols left

File: client-app/ui-kit/components/atoms/text-area/vc-text-area.vue [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="relative">
+  <div class="flex flex-col space-y-1">
```
Comment (goldenmaya) [inline client-app/ui-kit/components/atoms/text-area/vc-text-area.vue:2]: no need to use **flex & flex-col** here

File: client-app/ui-kit/components/atoms/text-area/vc-text-area.vue (hunk 1 above)
Comment (goldenmaya) [inline client-app/ui-kit/components/atoms/text-area/vc-text-area.vue:12]: better use <div> instead of <p>

Comment (ilyawzrd) [reply client-app/ui-kit/components/atoms/text-area/vc-text-area.vue]: done

File: client-app/ui-kit/components/atoms/text-area/vc-text-area.vue (hunk 2 above)
Comment (ilyawzrd) [reply client-app/ui-kit/components/atoms/text-area/vc-text-area.vue]: done

**Review by goldenmaya — APPROVED** [review]

**Pull request merged**